### DISCUSSION TITLE
Is unstable_batchedUpdates still needed?

### DISCUSSION THREAD

**Discussion by andrewblum** [excalidraw/excalidraw#8089] [General]
I noticed `unstable_batchedUpdates` in the `reactUtils` file. My understanding was this was no longer needed in React 18 and would be eventually removed from React.

**Comment by ad1992** [comment]
@andrewblum no its not needed since react 18 but we still would like to support the previous version (17) for the npm package and thats why we have still kept it. 
I think we can safely remove it once react 19 is out as everyone would have migrated to at least react 18 by then.
